### PR TITLE
feat(aws_utils): add MetricBuilder, add namespace

### PR DESCRIPTION
**Pull request by ConjunctiveNormalForm**

### CODE REVIEW

File: src/aws_utils/cloudwatch_utils.rs [hunk 1]
```diff
@@ -1,11 +1,53 @@
 pub mod cloudwatch_utils {
+    pub enum DimensionName {
+        Executor,
+    }
+    impl AsRef<str> for DimensionName {
+        fn as_ref(&self) -> &str {
+            match self {
+                DimensionName::Executor => "Executor",
```
Comment (codyborn) [inline src/aws_utils/cloudwatch_utils.rs]: `"Executor"` should these be constants?

Comment (ConjunctiveNormalForm) [reply src/aws_utils/cloudwatch_utils.rs]: good call, converted all strs to constants

File: Dockerfile [hunk 2]
```diff
@@ -31,21 +31,14 @@ cp ./target/release/$APP_NAME /bin/server
...
-    appuser
-USER appuser
-
 # Copy the executable from the "build" stage.
 COPY --from=build /bin/server /bin/
 
 # Expose the port that the application listens on.
 EXPOSE 1559
+
+# Add Tini
```
Comment (codyborn) [inline Dockerfile:40]: Nit: can you add a quick explanation why we need this in the comment?

**Review by codyborn — APPROVED** [review]
🚢

**Pull request merged**